www/res/css/perun.css

```diff
@@ -0,0 +1,3 @@
+.btn {
+    color: white !important;
+}
```

Comment:
```suggestion
```
`.btn.btn-primary` should have text color set (to background color), if not, just add CSS class `text-light` or `text-white`, you don't need a separate CSS file.

Reply:
I am afraid these classes are not available in Bootstrap v3. However, we might want to define them in the standard CSS file. I will need to check the other CSS files if such a class is available.

Reply:
As I thought. Both BS3 and SSP default CSS do not contain any class for styling links to such colours. Perhaps we can then define a default CSS file with such utility classes?